odoo/cli/help.py

```diff
@@ -0,0 +1,28 @@
+from .command import Command, commands, load_addons_commands, load_internal_commands
+
+import odoo
+
+
+class Help(Command):
+    """ Display the list of available commands """
+
+    def run(self, args):
+        load_internal_commands()
+        load_addons_commands()
+
+        print(  # noqa: T201
```

Comment:
I liked it better when there was a static string at the top of the file.

Reply:
![immagine](https://media1.tenor.com/m/T3H92Qstl68AAAAd/p-ong.gif)
Lemme know

Reply:
Done.